src/styles/index.scss

```diff
@@ -1,3 +1,84 @@
-body {
+@import './utils/variables';
+
+* {
   margin: 0;
+  box-sizing: border-box;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+@mixin center {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+// Mixin to style the day cell
+@mixin day-cell-style($bg-color, $border-color) {
+  height: $day-cell-height;
+  width: $day-cell-width;
+  border: 1px solid $border-color;
+  background-color: $bg-color;
+  transition-duration: 500ms;
+}
+
+@mixin day-cell-hover-style($bg-color) {
+  background-color: $bg-color;
+  transform: translateY(-20px);
+  cursor: pointer;
+}
+
+// Function to calculate dynamic calendar width based on parameters
+@function calendar-width($day-cell-width, $cols-gap-size, $calendar-padding) {
+  @return ($day-cell-width * 7) + ($cols-gap-size * 6) + (2 * $calendar-padding);
+}
```

Comment:
Great use of SCSS function to calculate the dynamic calendar width! It's a good practice to use SCSS functions for complex calculations. This makes your code cleaner and easier to understand.